bouncer/src/repo/manipulator/modelconvertor/import/repo_model_import_3drepo.cpp

```diff
@@ -432,7 +462,9 @@ bool RepoModelImport::importModel(std::string filePath, uint8_t &err)
 		repoInfo << "Loading BIM file [VERSION: " << incomingVersion << "]";
 
 		size_t metaSize = REPO_VERSION_LENGTH + sizeof(fileMeta);
-		if (incomingVersion == REPO_V3)
+		//TODO: Clean up the version string/number/comparison stuff into a class
+		uint8_t incomingVersionNo = std::stoi(incomingVersion.substr(3,3));
+		if (incomingVersionNo > 2)
 		{
 			fin->read((char*)&file_meta, REPO_V3_FILEMETA_BYTE_LEN);
```

Comment:
maybe just rewrite this to a static dictionary?

as `REPO_V3_FILEMETA_BYTE_LEN` is also the same as V3's byte length, the naming can be confusing to people.

i.e. 
```suggestion
// Have this array declared somewhere (maybe next to the min version supported variable)
const std::unordered_map<int, int> FILE_META_BYTE_LEN_BY_VERSION = {{2, 56}, {3, 72}, {4, 72}};

// Then you no longer need an if and this line becomes:
			fin->read((char*)&file_meta, FILE_META_BYTE_LEN_BY_VERSION [incomingVersionNo]);
```

You could even integrate this with the supported version check - if there's no entry in the dictionary for that number then it's an unsupported version.